src/main.js

```diff
@@ -44,16 +45,15 @@ let load = base.loader({
         referencePrefix:  'taskcluster-treeherder/v1/exchanges.json',
         publish:          cfg.app.publishMetaData,
         aws:              cfg.aws,
-        drain:            drain,
-        component:        cfg.app.statsComponent,
+        monitor: monitor.prefix('publisher'),
         process,
```

Comment:
I didn't see it complain about this, I was just copying what other projects were doing.  if it's not needed I omit it.